services/datalake/pod-datalake/schema/datalake.sql

```diff
@@ -0,0 +1,39 @@
+
+CREATE SCHEMA IF NOT EXISTS blob;
+
+-- B L O B
+
+CREATE TABLE IF NOT EXISTS blob.location AS ENUM ('eu', 'weur', 'eeur', 'wnam', 'enam', 'apac');
```

Comment:
Shouldn't we use `CREATE TYPE` instead of `CREATE TABLE` here?

Reply:
thanks, fixed